VaibhavCodeClub/learn
# Added details to Shapesandlines module Issue no #40

This PR adds a new feature to the app, introducing a Shapes and Lines page. Users can now explore various shapes and lines along with their visual representations.

Changes Made:

    Created a new page named ShapesPage to display shapes and lines.
    Added SVG images for shapes and lines to the assets folder.
    Implemented navigation functionality to cycle through different shapes and lines.
    Integrated text-to-speech functionality for pronouncing the names of shapes and lines.
    Styled the page with a colorful background and centered layout for better visualization.
    Added navigation drawer for easy access to other pages.

How to Test:

    Navigate to the Shapes and Lines page from the app drawer.
    Explore different shapes and lines by tapping the navigation buttons.
    Listen to the pronunciation of each shape and line by tapping the volume icon.
    Verify that the page layout and functionality work as expected on different screen sizes.
